pcc/src/cli/commands/login.ts

```diff
@@ -0,0 +1,88 @@
+import { OAuth2Client } from 'google-auth-library';
+import http from 'http';
+import url from 'url';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import open from 'open';
+import destroyer from 'server-destroy';
+import { readFileSync } from 'fs';
+import nunjucks from 'nunjucks';
+nunjucks.configure({ autoescape: true });
+
+const OAUTH_SCOPES = [
+  'https://www.googleapis.com/auth/drive.file',
```

Comment:
Why do we need this scope?

Reply:
I just included all scopes which are required for Add On except Add on specific ones.
But yeah, this is not used atm.